recipes-ni/sysconfig-settings/sysconfig-settings_1.0.bb

```diff
@@ -57,7 +63,39 @@ SRC_URI_append = "file://uixml/nilinuxrt.sshd_enable.binding.xml \
                   file://uixml/nilinuxrt.sshd_enable.def.xml \
 "
 
-FILES_${PN}-ssh += "/usr/local/natinst/share/uixml/sysconfig/nilinuxrt.sshd_enable.*"
+FILES_${PN}-ssh = "/usr/local/natinst/share/uixml/sysconfig/nilinuxrt.sshd_enable.*"
+
+# sysconfig-settings-ui package
+PACKAGES += "${PN}-ui"
+
+SUMMARY_${PN}-ui = "System configuration files to enable UI"
+DESCRIPTION_${PN}-ui = "Configuration files to enable UI for the National Instruments System Configuration subsystem."
+
+SRC_URI_append = "file://systemsettings/ui_enable.ini \
+                  file://uixml/nilinuxrt.System.binding.xml \
+                  file://uixml/nilinuxrt.System.const.xml \
+                  file://uixml/nilinuxrt.System.const.de.xml \
+                  file://uixml/nilinuxrt.System.const.fr.xml \
+                  file://uixml/nilinuxrt.System.const.ja.xml \
+                  file://uixml/nilinuxrt.System.const.ko.xml \
+                  file://uixml/nilinuxrt.System.const.zh-CN.xml \
+                  file://uixml/nilinuxrt.System.def.xml \
```

Comment:
Are these actually a part of the embedded UI checkbox? They look like general sysconfig values.

@gratian ?

Reply:
Seems like these are Locale settings dependent on Embedded UI (but it doesn't look like they need to be part of this package) and [they all](https://github.com/ni/meta-nilrt/commit/4da123d6fa63300d32eeea0bcd978491d38c1497#diff-bf24eed8d73b855770e6be4f58c2c98e80f52cb45acb71637141f3f31a989b80) have this comment

> Delete this file once we are able to support localization for UI Embedded Smasher Targets

Do we support localization for UI Embedded Smasher Targets now?

[Ref](https://nio365.sharepoint.com/sites/labview-real-time-pse-escalations/SitePages/Enable%20Embedded%20UI%20checkbox%20disabled%20for%20non-English%20Locale%20on%20903x.aspx)

Reply:
These files were added in https://review-board.natinst.com/r/66727 and are all related to Embedded UI. I created [another story](https://ni.visualstudio.com/DevCentral/_workitems/edit/1821122) to figure out if these are still required. I'd like us to move forward with these changes for now.